addons/sale_project/models/project.py

```diff
@@ -366,7 +366,7 @@ def _get_sale_order_items_query(self, domain_per_model=None):
             ])
         project_query = self.env['project.project']._where_calc(project_domain)
         self._apply_ir_rules(project_query, 'read')
-        project_sql = project_query.select('id', 'sale_line_id')
+        project_sql = project_query.select(f'{self._table}.id ', 'sale_line_id')
```

Comment:
That's a strange issue you got. :o 

maybe we should also add the table alias on `sale_line_id` to be sure we will no longer that strange issue. 🤷🏼‍♂️ 

What do you think? 🙂